src/main/java/sirius/search/Query.java

```diff
@@ -1017,6 +1017,7 @@ public SearchResponse queryRaw() {
         if (Microtiming.isEnabled()) {
             w.submitMicroTiming("ES", "RAW: " + toString(true));
         }
+        indexAccess.addSelectTiming(w.elapsedMillis());
```

Comment:
wouldnt it make more sense to do this in SearchRequestBuilder.execute().actionGet() ?